leaderboard-ui/src/app/landing-page/landing-page.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="container page-container">
+    <div class="container title-container">
+        <div class="container logo-container">
+            <img src="../../assets/images/ix-logo.png" alt="Index Exchange" width="92px" height="76px">
+        </div>
+        <div class="container heading-container">
+            <h1>IX House Cup Leaderboard</h1>
+        </div>
+    </div>
+    <div class="container house-leaderboard-container">
+        <app-team-leaderboard></app-team-leaderboard>
```

Comment:
and anything that propagates asynchronously from `ngOnChanges` will trigger a change detection cycle in the template. So the template will look something like:

```
<ng-container *ngIf="this.userData">
    <ng-container *ngFor="let user of userData | async">
        <ng-container">
             {{ user.whatever }}
         </ng-container>
     </ng-container>
<ng-container>

```

Reply:
Just realized this is a different template from the controller I commented on lol - but you get the idea 🤙

Reply:
coudln't get async to work properly D= will attempt if we get time